source/style-guide/style/metadata.txt

```diff
@@ -0,0 +1,19 @@
+.. _metadata:
+
+=============
+Metadata
+=============
+
+.. default-domain:: mongodb
+
+.. include:: /includes/styles/corrections.rst
+
+Use ``.. meta::`` to add meta tags to a page. Adding meta tags aids in 
+searches both internally and via the Internet.
+
+.. code-block:: rst
+     
+   .. meta::
+      :description: read concern, local read concern, read isolation, 
+      transactions, multi-document transactions 
```

Comment:
Do we have any information on which meta tags are available and when to use each one? Even a link to that info would be useful here.